pihole/MANIFEST

```diff
@@ -28,30 +28,20 @@
 			"Type": 10,
 			"Hash": "0000000000000000000000000000000000000000000000000000000000000000"
 		},
-		{
-			"Name": "pihole_rtypes",
-			"Type": 1,
-			"Hash": "0000000000000000000000000000000000000000000000000000000000000000"
-		},
-		{
-			"Name": "pihole_rcodes",
-			"Type": 1,
-			"Hash": "0000000000000000000000000000000000000000000000000000000000000000"
-		},
 		{
 			"Name": "pihole_status_codes",
 			"Type": 1,
-			"Hash": "0000000000000000000000000000000000000000000000000000000000000000"
+			"Hash": "fa3a39987aab09e29029ceba10d1d1d48880eb2f449dc3b2748c329696e95382"
```

Comment:
Please kill the hashes when submitting PRs
```
sed -i 's/"Hash": ".*"/"Hash": "0000000000000000000000000000000000000000000000000000000000000000"/g' MANIFEST
```

Reply:
@kris-watts-gravwell zeroed out hashes and added the requested tag macro